### ISSUE TITLE
Composable function naming

### ISSUE THREAD

**Issue by Azamat989** [pinterest/ktlint#2139]
## Expected Behavior
`function-naming` rule must allow (or even require?) composable function name start with uppercase letter.
https://developer.android.com/kotlin/style-guide#function_names

```kotlin
@Composable
fun NameTag(name: String) {
    // …
}
```

## Your Environment

* Version of ktlint used: 0.50
* Operating System and version: mac

**Comment by paul-dingemans** [comment]
Ktlint will not take annotations like `@Composable` into account.

In case your project is using this kind of annotation, it might be better to [disable the rule for your project](https://pinterest.github.io/ktlint/0.50.0/faq/#how-do-i-enable-or-disable-a-rule).

**Comment by Azamat989** [comment]
This is sad. @Composable annotation will be widely used in the near future by android developers.

**Comment by paul-dingemans** [comment]
You can always [write your own custom rule](https://pinterest.github.io/ktlint/0.50.0/api/custom-rule-set/). Or have a look at https://twitter.github.io/compose-rules/ktlint/
